pravega/zookeeper-operator
# Issue 220: Added support for more zookeeper configuration options

Signed-off-by: anisha.kj <anisha.kj@dell.com>

### Change log description
Added more zookeeper configuration options in zookeeper configmap.

### Purpose of the change

Fixes #220, #224 

### What the code does

Added more options as part of zookeeper spec under config, so that users are able to configure these values
### How to verify it

Able to set the configuration options, and pod should come up with the values set.
